fluxx.js

```diff
@@ -55,15 +55,20 @@ define([
         this.KEEPERS_SPRITES_PATH = g_gamethemeurl + "img/keepers.png";
         this.KEEPERS_SPRITES_PER_ROW = 10;
 
-        this.CARDS_TYPES = {
+        var creeperPackOffset = 19 + 30 + 27 + 23 + 3; // all base game cards + Basic Rules / Back / Front
```

Comment:
This might work for now. 
But I think we should simply add all the cards (including the new ones) in the one spritesheet?
I'll merge as is but will think about it.